data/meson.build

```diff
@@ -43,13 +43,11 @@ appdata_file = i18n.merge_file(
   install_dir: datadir / 'metainfo'
 )
 # Validate Appdata
-if appstream_util.found()
+if appstreamcli.found()
   test(
-    'validate-appdata', appstream_util,
-    args: [
-      'validate', '--nonet', appdata_file.full_path()
-    ],
-    depends: appdata_file,
```

Comment:
Is it no longer necessary?

Reply:
"appstream-glib and appstream-util are out-of-date, and GNOME developers suggest using appstreamcli to validate appdata.

I'm in the process of migrating GNOME and Circle apps to appstreamcli. It also provides better support for AppStream 1.0, which was published one month ago.

- https://gitlab.gnome.org/World/eartag/-/merge_requests/95#note_1874726
- https://gitlab.gnome.org/GNOME/Initiatives/-/issues/46

Reply:
Oh sorry, I meant the `depends: appdata_file`

Reply:
I see several version of appdata validation. I have no idea how to use them: https://mesonbuild.com/Release-notes-for-0-46-0.html#test-now-supports-the-depends-keyword-argument

```
  appstreamcli = find_program('appstreamcli', required: false)
  if appstreamcli.found()
    test(
      'validate-appdata', appstreamcli,
      args: [
        'validate', '--no-net', '--explain', appdata_file
      ],
      depends: appdata_file,
    )
  endif
```


```
appstreamcli = find_program('appstreamcli', required: false)
if appstreamcli.found()
  validate_args = ['validate', '--no-net', appstream_file]
  test('Validate appstream file', appstreamcli, args: validate_args)
endif
```

```
appstreamcli = find_program('appstreamcli', required: false)
if (appstreamcli.found())
  test('Validating appdata',
    appstreamcli,
    args: ['validate', '--no-net', appstream_file],
    workdir: meson.current_build_dir()
  )
endif
```

Reply:
I also have little idea of how it works since it is simply built from `gtk-rust-template`. I think I would keep `depends: appdata_file` for now